charts/openfga/values.schema.json

```diff
@@ -61,6 +61,9 @@
                 "uri": {
                     "type":["string", "null"]
                 },
+                "uriSecret": {
+                    "type":["string", "null"]
```

Comment:
Please add an appropriate description here. It should include details about what the sub-contract on the 'uri' key.

Reply:
All done! Let me know if the description is appropriate and if there is anything else I need to do.